profiles/development.cfg

```diff
@@ -6,6 +6,7 @@ extends =
 auto-checkout =
     recensio.plone
 parts +=
+    instance
```

Comment:
```suggestion
```

It seems to me like it is not needed:
```
[ale@flo recensio.buildout]$ ln -sf profiles/development.cfg buildout.cfg 
[ale@flo recensio.buildout]$ cat custom.cfg 
[buildout]
extensions += mr.developer
auto-checkout =
    recensio.plone


[sources]
recensio.plone = git https://github.com/syslabcom/recensio.plone.git
[ale@flo recensio.buildout]$ ./bin/develop rb|grep instance
INFO: Running buildout.
...
Installing instance.
...
```

Reply:
strange. i do not get `instance` generated without this change. do you have any modifications in `profiles/development.cfg`?

I use a custom requirements:
```
$ cat local-requirements.txt 
pdbpp
Products.PDBDebugMode
plone.reload
Products.PrintingMailHost
profilehooks

-r requirements.txt
```

and my custom.cfg is:
```
[buildout]

[instance]
environment-vars +=
    ENABLE_PRINTING_MAILHOST True
zcml-additional +=
  <configure xmlns="http://namespaces.zope.org/zope"
             xmlns:plone="http://namespaces.plone.org/plone">
  <plone:CORSPolicy
    allow_origin="http://localhost:3000,http://0.0.0.0:3000,http://localhost:3001,http://0.0.0.0:3001"
    allow_methods="DELETE,GET,OPTIONS,PATCH,POST,PUT"
    allow_credentials="true"
    expose_headers="Content-Length,X-My-Header"
    allow_headers="Accept,Authorization,Content-Type,X-Custom-Header,Origin"
    max_age="3600"
    />
  </configure>

[omelette]
ignore-develop = True
eggs += recensio.plone[test]

[scripts]
eggs += recensio.plone[test]
```

Reply:
Nope:
```
[ale@flo recensio.buildout]$ git status 
On branch plone6
Your branch is up to date with 'origin/plone6'.

nothing to commit, working tree clean

[ale@flo recensio.buildout]$ git show|cat
commit 9e4ee88b1845f1eeb79bf4c25974599e1a5daac0
Merge: 9251aad 22994a2
Author: Johannes Raggam <thetetet@gmail.com>
Date:   Fri Feb 3 11:26:05 2023 +0100

    Merge pull request #18 from syslabcom/longrequests
    
    Monitor long requests
```

Reply:
This is really strange, even if I use your `custom.cfg`, I have the instance script:
```
[ale@flo recensio.buildout]$ cat > custom.cfg 
[buildout]

[instance]
environment-vars +=
    ENABLE_PRINTING_MAILHOST True
zcml-additional +=
  <configure xmlns="http://namespaces.zope.org/zope"
             xmlns:plone="http://namespaces.plone.org/plone">
  <plone:CORSPolicy
    allow_origin="http://localhost:3000,http://0.0.0.0:3000,http://localhost:3001,http://0.0.0.0:3001"
    allow_methods="DELETE,GET,OPTIONS,PATCH,POST,PUT"
    allow_credentials="true"
    expose_headers="Content-Length,X-My-Header"
    allow_headers="Accept,Authorization,Content-Type,X-Custom-Header,Origin"
    max_age="3600"
    />
  </configure>

[omelette]
ignore-develop = True
eggs += recensio.plone[test]

[scripts]
eggs += recensio.plone[test]
[ale@flo recensio.buildout]$ ./bin/develop rb|grep instance
INFO: Running buildout.
...
Uninstalling instance.
Installing instance.
Generated script '/home/ale/Code/buildouts/recensio.buildout/bin/instance'.
...
```